<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>741</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>    查找信息</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>学号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>       姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>        专业</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>         班级</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>            性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>               年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_refresh">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_ch1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ch2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ch3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ch4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ch5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ch6"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_change">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>    修改信息</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>    插入信息</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>    删除信息</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_ser1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ser2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ser3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ser4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ser5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ser6"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_find">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_add1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_add2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_add3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_add4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_add5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_add6"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>插入</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>一键排序</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_del1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_del2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_del3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_del4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_del5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_del6"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_return2">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
